modules/cnf-configuring-high-priority-workload-pods.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-configuring-high-priority-workload-pods_{context}"]
+= Disabling power saving mode for high priority pods
+
+You can configure pods to ensure that high priority workloads are unaffected when you configure power saving for the node that the workloads run on.
+
+When you configure a node with a power saving configuration, you must configure high priority workloads with performance configuration at the pod level, which means that the configuration applies to all the cores used by the pod.
+
+By disabling P-states and C-states at the pod level, you can configure high priority workloads for best performance and lowest latency.
+
+.Configuration for high priority workloads
+[cols="1,2,3", options="header"]
+
+|===
+| Annotation | Possible Values | Description
+
+|`cpu-c-states.crio.io:` a|  * `"enable"`
+* `"disable"`
+* `"max_latency:microseconds"` | This annotation allows you to enable or disable C-states for each CPU. Alternatively, you can also specify a maximum latency in microseconds for the C-states. For example, enable C-states with a maximum latency of 10 microseconds with the setting `cpu-c-states.crio.io`: `"max_latency:10"`. Set the value to `"disable"` to provide the best performance for a pod.
+
+| `cpu-freq-governor.crio.io:` | Any supported `cpufreq governor`. | Sets the `cpufreq` governor for each CPU. The `"performance"` governor is recommended for high priority workloads.
+|===
+
+.Prerequisites
+
+* You have configured power saving in the performance profile for the node where the high priority workload pods are scheduled.
+
+.Procedure
+
+. Add the required annotations to your high priority workload pods. The annotations override the `default` settings.
++
+.Example high priority workload annotation
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  ...
```

Comment:
"For [snippets or sections of a file](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks)... for YAML, use ('#…​')" throughout this code block

```suggestion
# ...
```